client/src/components/Canvas/Canvas.js

```diff
@@ -1,3 +1,28 @@
-export const Canvas = (props) => {
-  return <canvas id="canvas" width="700" height="700" />;
+import React, { useState, useEffect } from 'react';
+import { fabric } from 'fabric';
+import { Circle } from '../Shapes';
+
+const Canvas = (props) => {
+  const [canvas, setCanvas] = useState('');
+
+  useEffect(() => {
+    setCanvas(initCanvas());
+  }, []);
+
+  const initCanvas = () =>
+    new fabric.Canvas('canvas', {
+      height: 700,
+      width: 700,
```

Comment:
I wonder if there's a way to set there dynamically based on the size of the screen